ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/Input Variables.md

```diff
@@ -470,6 +470,7 @@ daemon_terminattr:
   ipfixaddr: <str>
   sflow: <bool>
   sflowaddr: <str>
+  cvconfig: <bool>
```

Comment:
did you add this by hand? You should run the molecule `molecule converge -s build_schemas_and_docs` locally to auto generate everything from the fragment file.